app.rb

```diff
@@ -0,0 +1,96 @@
+require "pry"
+
+# Game welcome
+puts "Welcome to Gametime"
+puts "Type cardlist.print_details to view all flashcards"
+puts "Type edit to change flashcard"
+puts "delete to delete a flashcard"
+puts "view to see score"
+puts "Type card (plus number)to play"
+user_input = gets.chomp
+
+# Flashcard board
+$cardlist = []
+$cards = [card1, card2, card3, card4]
+userinput = ""
+
+class Flashcard
+  attr_accessor :front, :back
+
+  def initialize(front, back)
+    @front = front
+    @back = back
+  end
+
+# prompt user to create cards
+  def card
+    card = @card
```

Comment:
You haven't defined `@cards` yet. You could do that in your constructor (in your `initialize` method) however.

```rb
  @cards = {front: front, back: back}
```